resources/views/admin/post/index.blade.php

```diff
@@ -0,0 +1,83 @@
+@extends('layouts.admin.base')
+@section('admin_content')
+    @include('layouts.components.notification')
+    <div class="table-agile-info">
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                {{ __('admin.title_list_post') }}
+            </div>
+            <div class="row w3-res-tb">
+                <div class="col-sm-5 m-b-xs">
+                </div>
+                <div class="col-sm-4">
+                </div>
+                <div class="col-sm-3">
+                    <form class="input-group" action="">
+                        <input type="text" class="input-sm form-control" name="key">
+                        <span class="input-group-btn">
+                            <button class="btn btn-sm btn-default" type="submit">Search</button>
+                        </span>
+                    </form>
+                </div>
+            </div>
+            <div class="table-responsive">
+                <table class="table table-striped b-t b-light" id="myTable">
+                    <thead>
+                        <tr>
+                            <th>{{ __('admin.number') }}</th>
+                            <th>{{ __('admin.image') }}</th>
+                            <th>{{ __('admin.title') }}</th>
+                            <th>{{ __('admin.content') }}</th>
+                            <th>{{ __('admin.category') }}</th>
+                            <th>{{ __('admin.status') }}</th>
+                            <th style="width:75px;"></th>
+                        </tr>
+                    </thead>
+
+                    <tbody>
+                        @foreach ($Posts as $index => $post)
+                            <tr>
+                                <td>{{ ++$index }}</td>
+                                <td><img src="{{ Storage::url($post->image) }}" height="100" width="100"></td>
+                                <td>
+                                    <h4>{{ $post->title }}</h4>
+                                </td>
+                                <td>{!! nl2br(e(Str::limit($post->content, 100))) !!}</td>
+                                <td>
+                                    @foreach ($post->categories as $category)
+                                        {{ $category->name }}
+                                        <br>
+                                    @endforeach
+                                </td>
+                                <td>
+                                    <span class="text-ellipsis">
+                                        @if ($post->status === \App\Models\Post::APPROVED)
```

Comment:
<a href="{{ route('posts.update.status', ['id' => $post->id]) }}">
<span class="text-approve {{$post->status === \App\Models\Post::APPROVED ? 'approved' : 'not-approved'}}">{{ $post->status_name }}</span>
</a>